src/photos/components/PhotosViewer.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
```

Comment:
This is very similar to `FileViewer`, can't we factorize a lot of code?

Reply:
See ^^